modules/Twitter/src/main/java/totetmatt/gephi/twitter/utils/UserIdResolver.java

```diff
@@ -0,0 +1,27 @@
+package totetmatt.gephi.twitter.utils;
+
+import java.io.IOException;
+import java.util.Iterator;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class UserIdResolver {
+
+    static public long resolve(String userScreenName) throws IOException {
+        Document doc;
+        doc = Jsoup.connect(
+                "http://www.twitter.com/" + userScreenName.toLowerCase()).get();
+        Elements elems = doc.getElementsByAttribute("data-user-id");
+        Iterator<Element> els = elems.iterator();
+        while (els.hasNext()) {
+            Element el = els.next();
+            if (!el.attr("data-user-id").isEmpty()) {
+                return Long.parseLong(el.attr("data-user-id"));
+            }
+        }
+
+        return 0;
+    }
+}
```

Comment:
Out of curiosity, why are you directly accessing the HTML of a Twitter page rather than using the REST API? The [GET users/lookup](https://dev.twitter.com/rest/reference/get/users/lookup) endpoint is designed to for this username to numeric userid conversion and can also handle batches of up to 100 usernames (rather than requiring a separate request for each username)